flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/CatalogPropertiesUtil.java

```diff
@@ -176,30 +211,32 @@ private static List<String> deserializePartitionKeys(Map<String, String> map) {
         return partitionKeys;
     }
 
-    private static Schema deserializeSchema(Map<String, String> map) {
+    private static Schema deserializeSchema(Map<String, String> map, String schemaKey) {
```

Comment:
Why change this method? is it necessary?

Reply:
It's meant to extract schema by the fallback key in the hive in [link](https://github.com/apache/flink/blob/e818c11d104623f47b7568e39759a285a357a1e9/flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java#L773)

Reply:
Make sense.